src/odin/util.cc

```diff
@@ -134,5 +135,29 @@ const std::unordered_map<std::string, std::string>& get_locales_json() {
   return locales_json;
 }
 
+std::string turn_lane_direction(uint16_t turn_lane) {
+  switch (turn_lane) {
+    case baldr::kTurnLaneReverse:
+      return "uturn";
```

Comment:
The turn direction strings used here are repeated in 4 different places in the route serializer. Wonder if we can create constants that can be reused in multiple places?

If its trivial to do, I'd suggest doing it in this PR, else we can open a follow-up task to consolidate the various string constants used in `odin` and other modules.

Reply:
@mandeepsandhu I added a new header in https://github.com/valhalla/valhalla/pull/2675/commits/7485b30c5346f0654dae382de76450e5d1b32c58 what do you think of this approach?